documentation/4.0/content/managing-domains/domain-resource.md

```diff
@@ -206,8 +206,15 @@ The elements `serverStartPolicy`, `serverPod` and `serverService` are repeated u
 Elements related to the customization of liveness and readiness probes:
 * See [Liveness probe customization]({{< relref "/managing-domains/domain-lifecycle/liveness-readiness-probe-customization#liveness-probe-customization" >}}) for details about the elements related to liveness probe customization and [Readiness probe customization]({{< relref "/managing-domains/domain-lifecycle/liveness-readiness-probe-customization#readiness-probe-customization" >}}) for details about the elements related to readiness probe customization.
 
+Cluster spec elements:
+
+For a complete list of the Cluster spec elements, see [Cluster Spec](https://github.com/oracle/weblogic-kubernetes-operator/blob/{{< latestMinorVersion >}}/documentation/domains/Cluster.md#cluster-spec). The following paragraph has additional context information for the `clusterServer.sessionAffinity` element.
+* `sessionAffinity`: This is an advanced setting that is only applicable when the `kube-proxy` is running in the non-default proxy modes such as [User space (legacy) proxy mode](https://kubernetes.io/docs/concepts/services-networking/service/#proxy-mode-userspace) and [IPVS proxy mode](https://kubernetes.io/docs/concepts/services-networking/service/#proxy-mode-ipvs). It is used to enable session affinity based on the client's IP addresses. See the [Virtual IPs and service proxies](https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies) for more information. Must be `ClientIP` or `None`. Defaults to `None`.
```

Comment:
only applicable -> applicable only
in the non-default proxy modes such as -> in non-default proxy modes, such as
For more information, see the [Virtual IPs and service proxies](https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies).

Reply:
Changed.